spec/unit/config_spec.rb

```diff
@@ -7,6 +7,7 @@
   it { is_expected.to respond_to(:statefile) }
   it { is_expected.to respond_to(:namespace) }
   it { is_expected.to respond_to(:minter_class) }
+  it { is_expected.to respond_to(:identifier_in_use) }
```

Comment:
Isn't this a bit duplicative? You are explicitly testing this method below?

Reply:
I wouldn't normally include this, but I was following the existing pattern.  Methods template and minter_class are also explicitly tested and have the respond_to test.